composer.json

```
@@ -56,9 +56,9 @@
     "oscarotero/env": "^2.1",
     "roots/bedrock-autoloader": "^1.0",
     "roots/bedrock-disallow-indexing": "^2.0",
-    "roots/wordpress": "6.0.3",
-    "roots/wp-config": "1.0.0",
-    "roots/wp-password-bcrypt": "1.1.0",
+    "roots/wordpress": "^6.0.3",
```

setting this to `"^6.0.3"` will mean that we need to manually update the `composer.json` again for WP 6.1 in a couple weeks. I don't know that there's a reason to not just let WordPress updates happen and do this instead since `"prefer-stable"` is true:

```suggestion
    "roots/wordpress": "*",
```

---

This is true too - thoughts on @kporras07 [concerns](https://github.com/pantheon-systems/wordpress-composer-managed/pull/36#discussion_r998424049)? Specifically:

> I'm not comfortable with "*" as a pretty messed up composer conflict could even downgrade the site instead of upgrading it.

---

^6.0.3 means everything between 6.0.3 and 7.0.0 (not included).

~6.0.3 means that @jazzsequence said.

Thanks composer for being confusing 😬

---

Let's go with `^6.0.3` for now so we unblock John.  

@jspellman814 can you start a thread in our squad channel around best practices here versus when to use a `^` or `*`?  A good outcome from the conversation would be to add our decision as a page in https://github.com/pantheon-systems/Standards.